controllers/territory_controller/territory_controller.py

```diff
@@ -167,6 +169,16 @@ def record_captures(self) -> None:
     def is_dirty(self) -> bool:
         return self._log_is_dirty
 
+    def get_scores(self) -> Mapping[Claimant, int]:
+        POINTS_PER_TERRITORY = 2
```

Comment:
If this is a constant it should live at the top level. If it's not, just inline it.